include/clang/Interpreter/CppInterOp.h

```diff
@@ -286,7 +286,11 @@ namespace Cpp {
 
   /// Gets a list of all the Methods that are in the Class that is
   /// supplied as a parameter.
-  CPPINTEROP_API std::vector<TCppFunction_t> GetClassMethods(TCppScope_t klass);
+  CPPINTEROP_API void GetClassMethods(TCppScope_t klass, std::vector<TCppFunction_t> &methods);
+
+  ///\returns Template function pointer list to add proxies for
+  /// un-instantiated/non-overloaded templated methods
```

Comment:
We need to document the parameters, too.

Reply:
done

Reply:
This is not resolved.

Reply:
done